src/game/renderer.cpp

```diff
@@ -343,7 +345,8 @@ void Renderer::renderNow()
 
 void Renderer::resizeEvent(QResizeEvent * event)
 {
-    resizeGL(event->size().width(), event->size().height());
+    if (m_rendererInitialized)
```

Comment:
I'd prefer {} always even with a single-line if, but doesn't matter here.

Reply:
I should have followed the style, I agree. Thanks for merging both PRs.